tests/basis_test.py

```diff
@@ -58,14 +27,16 @@ def test_cheb_call_degree_0_1():
         assert f0(i) == 1
         assert f1(i) == i
 
+
 def test_cheb_call_random_points():
     """Test chebyshev polynomial at some degree at some input"""
     numpy.random.seed(567)
-    ns = numpy.random.randint(20,size = 20)
+    ns = numpy.random.randint(20, size=20)
```

Comment:
This test should be parametrized on a few n, rather than generating a random list of n within the test. You won't need the for loop then.